srcpkgs/sane-airscan/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'sane-airscan'
+pkgname=sane-airscan
+version=0.99.24
+revision=1
+wrksrc="sane-airscan-${version}"
+build_style=gnu-makefile
+make_install_args="STRIP="
+hostmakedepends="pkg-config"
+makedepends="avahi-libs-devel sane-devel gnutls-devel libxml2-devel libjpeg-turbo-devel libpng-devel"
+short_desc="SANE driver for Apple AirScan"
```

Comment:
Could you make the description closer to what's used by upstream?

> Scanner Access Now Easy - universal driver for eSCL (Apple AirScan) and WSD

Reply:
:sparkles: